instrumentation/vertx/vertx-redis-client-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/vertx/v4_0/redis/VertxRedisClientAttributesExtractor.java

```diff
@@ -7,21 +7,31 @@
 
 import static io.opentelemetry.instrumentation.api.internal.AttributesExtractorUtil.internalSet;
 
+import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.api.common.AttributesBuilder;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import io.opentelemetry.instrumentation.api.internal.SemconvStability;
 import io.opentelemetry.semconv.incubating.DbIncubatingAttributes;
 import javax.annotation.Nullable;
 
 enum VertxRedisClientAttributesExtractor
     implements AttributesExtractor<VertxRedisClientRequest, Void> {
   INSTANCE;
 
+  private static final AttributeKey<String> DB_NAMESPACE = AttributeKey.stringKey("db.namespace");
+
   @Override
   public void onStart(
       AttributesBuilder attributes, Context parentContext, VertxRedisClientRequest request) {
-    internalSet(
-        attributes, DbIncubatingAttributes.DB_REDIS_DATABASE_INDEX, request.getDatabaseIndex());
+    if (SemconvStability.emitStableDatabaseSemconv()) {
+      attributes.put(DB_NAMESPACE, String.valueOf(request.getDatabaseIndex()));
+    }
```

Comment:
`db.namespace` should already be getting captured by `DbClientAttributesExtractor`

Reply:
this class implements 'AttributesExtractor', not 'DbClientAttributesExtractor'

Reply:
see https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11575/commits/a17b1d214eed2360ae4d29f592591a7ad2297530